e2e/plugin-typescript-e2e/vite.config.e2e.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference types="vitest" />
+import { defineConfig } from 'vite';
+import { tsconfigPathAliases } from '../../tools/vitest-tsconfig-path-aliases.js';
+
+export default defineConfig({
+  cacheDir: '../../node_modules/.vite/plugin-typescript-e2e',
+  resolve: {
+    alias: {
+      '@code-pushup/test-nx-utils': '../../test-nx-utils/index.js',
+      '@code-pushup/test-setup': '../../test-setup/index.js',
+      '@code-pushup/test-utils': '../../test-utils/index.js',
+    },
+  },
+  test: {
+    reporters: ['basic'],
+    testTimeout: 120_000,
```

Comment:
In #976 you reduced the timeouts well below 2 mins for all E2E tests. I'm guessing the TS plugin could have a much lower timeout.